lib/archive/unpack.go

```diff
@@ -74,50 +80,50 @@ func Unpack(op trace.Operation, tarStream io.Reader, filter *FilterSpec, root st
 			return err
 		}
 
-		op.Debugf("processing tar header: %s", header.Name)
-
+		op.Debugf("processing tar header: %#v", *header)
+		op.Debugf("using FilterSpec : (%#v)", *filter)
```

Comment:
It likely is not necessary. I included it here since anyone could be calling this function and the filterspec is integral to what is exected from this function. I have moved it out of the loop :)